_posts/2023-04-18-ai-based-assistant.md

```diff
@@ -0,0 +1,55 @@
+
+All these reviews are extremely helpful for holiday makers. They provide additional information, transparency and help in the decision-making process when going through a wealth of offers. To create a detailed and thus helpful rating, the holiday maker must take some time to evaluate the individual categories, including the location, surrounding area, rooms, food, entertainment, and the hotel in general. 
+
+***Submit a test review: Writing reviews will be easier and faster in the future***
+
+HolidayCheck is building an [AI assistant](https://holidaycheck.de/svc/hc-review-assistant/travel-escapes/welcome.html) to make it even easier for holiday makers to write reviews in the future. This is still a beta version to be tested by as many passionate travellers as possible.
+
+Therefore, we are calling on all of you to submit a test review using the HolidayCheck assistant with the beta version. The submitted reviews will not go online but will be used to make the functionality even better. All participants will be entered into a raffle: This is done automatically once you interact with the assistant, and it generates reviews for you.
+
+***Generative AI can help people find the right way to express themselves***
+
+This Beta tool was developed as a joint project between Hubert Burda Media and HolidayCheck Group. [Jean-Paul Schmetz](https://de.linkedin.com/in/jpschmetz), Chief Scientist at Hubert Burda Media and [Konark Modi](https://de.linkedin.com/in/konarkmodi), Vice President Technology at HolidayCheck are leading the development of this AI assistant and answer the most important questions here. Jean-Paul Schmetz explains how AI can help with writing reviews: “Just like a calculator can help people make sure they calculate correctly, Generative AI can help people find the right way to express themselves. We have known for years that most of our users are somewhat insecure about their writing skills, and it prevents them from sharing their experiences with the rest of our users. While a lot of people focus on the ability to generate fake content with Generative AI, we focus on its ability to assist users in writing reviews that reflect their experience”, says Jean-Paul Schmetz.
+
+We asked both experts for more information about the HolidayCheck review assistant. Here are their answers:
+
+***Why did you develop your own AI assistant for HolidayCheck?***
+
+We strive to reduce the barrier to entry for users wanting to write a review. One does not need to be an expert in any language to submit a review, nor does one have to spend hours finding the words to use or not use. The users can simply start speaking or typing and the assistant helps to write a proper narrative with just a few words.
```

Comment:
barrier-to-entry is one word

Reply:
"We strive to reduce the barrier-to-entry in writing a review" can be a shorter sentence

Reply:
One does not need to be a language expert to submit a review, nor does one have to spend hours finding the right words to use

Reply:
"The assistant comes handy. Just start speaking or typing and the assistant helps to write a proper narrative with just a few words." (to make it more attractive !)